using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -332,14 +496,14 @@ You can pass the `launchOptions` that were received upon opening the application
 **Syntax**
 
 ```swift
-func trackNotificationReceive(_ trackInfo: [String: String])
+static func trackNotificationReceive(withUserInfo userInfo: [AnyHashable: Any])
```

Comment:
Why are we updating the 2.x Swift syntax?

Reply:
my bad that was by mistake